libtransmission/rpc-server.cc

```diff
@@ -1058,4 +1244,7 @@ tr_rpc_server::~tr_rpc_server()
     TR_ASSERT(tr_amInEventThread(this->session));
 
     stopServer(this);
+
+    tr_free(this->bindAddress);
+    this->bindAddress = nullptr;
```

Comment:
Is this (zeroing out fields on destruction) something we're still doing?.. @ckerr

Reply:
I agree this is not strictly necessary, but clearing the pointer shouldn't harm anything.

Reply:
probably ought to be a unique_ptr or (since the type is hidden inside rpc-server.cc) a shared_ptr